docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/hemoglobin/index.rst

```diff
@@ -6,4 +6,290 @@ Hemoglobin Model
+
+    And :math:`F_1(x)` is a mirror gumbel distribution parameterized by :math:`\mu` and :math:`\sigma`
+
+.. todo::
+
+	Include mathematical formulas for the gamma and mirror Gumbel distributions. For now, see the R code in the `Data Description Tables`_ section for details.
+
+.. note::
+
+	As summarized by Nathaneil, for context (details perhaps to be moved to a separate documentation page), the steps for finding an ensemble distribution are:
+
+	1). Each of the two-parameter distributions is "fit" to the data in each location/year/age/sex group using the "method of moments." I.e. you just choose the distribution that has the same mean and standard deviation as the data in each group, so there's not actually any complicated curve fitting going on here.
+
+	2). The distribution F of hemoglobin is assumed to be a mixture distribution that also comes from a 2-parameter family (GBD calls this an ensemble), of the form
+	
+	F(x; m_i,s_i) = w_1 F_1(x; m_i,s_i) + ... + w_k F_k(x; m_i,s_i),
+
+	where
+
+	m_i = mean Hb in group i
+
+	s_i = standard deviation of Hb in group i
+
+	F_1,...,F_k are the distributions in the list above (gamma, mirror gamma,etc.), parameterized by mean m and standard deviation s
+	F is the mixture distribution for Hb, also parameterized by mean m and standard deviation s
+
+	w_1,...,w_k are the global weights of the distributions in the mixture, used in all groups
+
+	(I think the important thing to note here is that the distribution weights are global, not depending on the location/year/age/sex, whereas it is only the mean and standard deviation that vary across groups. This is not totally clear from your description above.)
+
+	3). The global weights w_1,...,w_k are found by solving an optimization problem that somehow chooses the "best" weights that match the data in all the groups simultaneously. According to the YLD appendix, this "best fit" is defined by taking anemia prevalence data into account. Whatever they did to solve this optimization problem, the weights they came up with are w_1 = 0.4 (for F_1 = gamma distribution) and w_2 = 0.6 (for F_2 = mirror gumbel distribution), with w_j = 0 for all the other distributions.
+
+3. Generation of ensemble distributions for each location/year/age/sex group
+
+    Because anemia thresholds depend on pregnancy status, hemoglobin distributions were modeled separately for pregnant and non-pregnant females. The pregnancy model was identical to the non-pregnancy model except that the mean and variance were adjusted by the adjustment factor. The prevalence of anemia in pregnant women and non-pregnant women were then weighted by the pregnancy rate and combined to estimate population anemia prevalence. See the table below for the exact adjustment factors used.
+
+      The pregnancy rate was represented as :math:`(ASFR + SB) * 46/52`, where :math:`ASFR` is the location- and age-specific fertility rate and :math:`SB` is the location-specific stillbirth rate.
```

Comment:
You explain this below, but it might be worth also adding a quick comment here explaining that 46/52 represents (40 weeks of pregnancy + 6 weeks of lactation) / (52 weeks in a year).

Reply:
Also, did you intend this paragraph to be indented farther than the previous paragraph, or is that a mistake?